devenv/docker/blocks/jaeger/docker-compose.yaml

```diff
@@ -1,8 +1,10 @@
   jaeger:
     image: jaegertracing/all-in-one:latest
     ports:
-      - "6831:6831"
+      - "6831:6831/udp"
       - "16686:16686"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
   # Additional loki to generate some traces
```

Comment:
i think this part is not needed.

explanation:

when using docker-for-mac, automatically (no extra-hosts needed), when you are INSIDE a docker-container, you can access the host computer by accessing the hostname `host.docker.internal`. so for example, if on my macbook i run something at port 4444, then from inside a docker-container i can do `curl http://host.docker.internal:4444` and it will access that thing on my macbook.

for linux, this does not happen automatically. if you add
```
extra_hosts:
  - "host.docker.internal:host-gateway"
```

then even on linux the same thing as on mac will happen.

so if you put this part into your docker-compose, you can be sure that both on macos and linux, from inside the container, accessing `host.docker.internal` will point to the host computer.

but my understanding is that in this case, we do not connect from inside the jaeger to the host computer, the data is sent from the host-computer into the docker-container using `localhost`. that should work ok without any host-docker-internal.

Reply:
Thanks. FIxed